### ISSUE TITLE
Error: Schema validation of HMC credentials file failed on element 'metrics ->verify_cert'

### ISSUE THREAD

**Issue by Charles1000Chen** [zhmcclient/zhmc-prometheus-exporter#145]
zhmc-prometheus-exporter: 0.6.0
zhmcclient: 0.31.0

Currently, the HMC is using self-signed cert, I added the "verify_cert: false" in the credential file "hmccreds_1.yaml" like below:

```yaml
metrics:
  hmc: x.x.x.x
  password: xxxxxxxxxxx
  userid: xxxxxxx
  verify_cert: false
```
The zhmc exporter reports the following error when start:

```
Error: Schema validation of HMC credentials file /opt/zhmc-prometheus-exporter/hmccreds_1.yaml failed on element 'metrics': Additional properties are not allowed ('verify_cert' was unexpected)

Failed validating 'additionalProperties' in schema['properties']['metrics']:
    {'additionalProperties': False,
     'description': 'The HMC host and credentials',
     'properties': {'hmc': {'description': 'Hostname or IP address of HMC',
                            'type': 'string'},
                    'password': {'description': 'Password of the HCM '
                                                'userid',
                                 'type': 'string'},
                    'userid': {'description': 'Userid used by the exporter '
                                              'to log on to the HMC',
                               'type': 'string'}},
     'required': ['hmc', 'userid', 'password'],
     'type': 'object'}

On instance['metrics']:
    {'hmc': 'x.x.x.x',
     'password': 'xxxxxxxxxx',
     'userid': 'xxxxxxxx',
     'verify_cert': False}
```

**Comment by Charles1000Chen** [comment]
As we have zhmc-prometheus-exporter 0.6.0 depend on zhmcclient 0.31.0, need update the zhmc-prometheus-exporter/zhmc_prometheus_exporter/schemas/hmccreds_schema.yaml to support "verify_cert" property.

**Comment by Charles1000Chen** [comment]
In zhmc-prometheus-exporter: 0.6.0, the first line "zhmcclient>=0.19.0 # Apache-2.0" in the requirements.txt needs be updated to "zhmcclient>=0.19.0,<=0.30.2 # Apache-2.0".

**Comment by Charles1000Chen** [comment]
https://github.com/zhmcclient/zhmc-prometheus-exporter/pull/146

**Comment by andy-maier** [comment]
Thanks for catching this one. I indeed missed this aspect of the zhmcclient change. I'd like to address it by tolerating the verify_cert element, though.

**Comment by andy-maier** [comment]
The PR was merged and will be in 0.6.1.
